sample/samplers_test.go

```diff
@@ -55,6 +60,97 @@ func TestWeighted(t *testing.T) {
 	}
 }
 
+func modelHelper(t testing.TB) model.BytePairEncoding {
```

Comment:
Can change this later if we want - going to merge so that this PR doesn't go stale